workflows/viralintegration.nf

```diff
@@ -89,18 +270,27 @@ workflow VIRALINTEGRATION {
     workflow_summary    = WorkflowViralintegration.paramsSummaryMultiqc(workflow, summary_params)
     ch_workflow_summary = Channel.value(workflow_summary)
 
+    methods_description    = WorkflowViralintegration.methodsDescriptionText(workflow, ch_multiqc_custom_methods_description)
+    ch_methods_description = Channel.value(methods_description)
+
     ch_multiqc_files = Channel.empty()
```

Comment:
This can be a little cleaner like:

```
ch_multiqc_files = channel1
   .mix(channel2)
   .mix(channel3)
....
```

(you don't have to keep doing the `ch_multiqc_files = `)

Reply:
Do you have an example of this in practice that we can borrow from?

Reply:
Yep, like [this](https://github.com/nf-core/differentialabundance/blob/17aec4fd0ab2c84b1d384932187797aa7ea1e962/workflows/differentialabundance.nf#L241)